save-plugins/fix-plugin/src/commonMain/kotlin/com/saveourtool/save/plugins/fix/FixPlugin.kt

```diff
@@ -219,6 +231,40 @@ class FixPlugin(
         return fixedTestCopyToExpectedFilesMap
     }
 
+    @Suppress("SAY_NO_TO_VAR")
+    private fun getSarifFile(
+        executionResultStdout: List<String>,
+        fixPluginConfig: FixPluginConfig,
+        testsPaths: List<Path>,
+        tmpDirName: Path,
+    ): Pair<Path, Boolean> {
+        var isSarifFileEmulated = false
+
+        val sarifFile = if (fixPluginConfig.actualFixSarifFileName != null) {
```

Comment:
Do we even need a case when we are given actual sarif file? Could there be such a case?

Reply:
of course, actual sarif file in our terms here - sarif file with fixes, which we need to apply to test files

the **expected** files in fix plugin provided via `.kt` files

Reply:
I think it more rare case, when sarif report provided via stdout, rather than via file

Reply:
> of course, actual sarif file in our terms here - sarif file with fixes, which we need to apply to test files
> 
> the **expected** files in fix plugin provided via `.kt` files

However, we can also support expected fixes by sarif https://github.com/saveourtool/save-cli/issues/500

Reply:
Ok, agree